docs/diagrams/CheckInCommand-activity-diagram/CheckInCommand-activity-diagram.xml

```diff
@@ -0,0 +1 @@
+<mxfile userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36" version="9.4.1" editor="www.draw.io" type="device"><diagram name="Page-1" id="58cdce13-f638-feb5-8d6f-7d28b1aa9fa0">7Vxbk6o4EP411j7NFuGqj6PHmT1Vu1VbNVV7eYwQJTtIqBiPur9+EwgIITisCuiMvgx0rnzdX6fThBlZs/X+lcIk/I0EKBqZRrAfWd9Gpgls0+V/hOSQScaGmQlWFAey0lHwhv9FUmhI6RYHaFOpyAiJGE6qQp/EMfJZRQYpJbtqtSWJqqMmcIVqgjcfRnXpnzhgoZQCwzgW/ILwKpRDjx1ZsID++4qSbSzHG5nWMv1lxWuY9yXrb0IYkF1JZM1H1owSwrKr9X6GIoFtDlvW7qWhtJg3RTFr08Aam4sJWDgOAIvAWHpPsocfMNqi/BHSibJDDg7vgeuB30xDto64DPBL/iCJKN8wSNkbg0yUL3EUzUhEaNrQmrvWM58Pr8woeUelEtsx+I+XpNCh4Nhpho4o8ska+/I6ggsUTQus855iEqfDkpi9wDWOhPX9gWgAYyjF0tSAKe9LczAMOQcY4VXMZT7HEPHCaYApNzJMhHBDtkKF0zrQOXKIMrQviSTwr4isEaMHXkWWcqvJmkiS5DaxO1qcJUVhydZyGZQ2vio6PuqZX0hVN6i9hZYDzgh5SygLyYrEMJofpSVdGVVb+AcxdpBIwy0jXHTs4VdCElmPQ0cPf4n2Pzv57d+yu2w+YhKNli1FXCPURw3mLP0QN8oVks1svVIoiiDDP6rD6TCWTX8nmE+kUKaVu4Zcma6ipmwKspWiqWIa7ZTX8JAlbb4iMTXuRtZiBtwZ8dZsIzwhphtRtCDkHcermtar/NuFmKG3BKbw7rirr+oZUl9qmYNaY7vwer6W7culZNqN8PsaXLar6gcaMheyMptVMzmHzSa0Fr7pGNBfGi7wgqfJGWwu6fU0+igOnsUSK3pKUJxJJO7jk+puqbdcPxFaslPaOUn8PGooEb+JN4qTucA1tFVYnbBDuN/zXKxdB9bpCbY8Orxf3Dpfdmw1OrjesmN/DD4NyXqx3Xy8cjS4Vw2IzR7XVRZcr+5xxxqHO76Cwx3fsR06df56ffH3nv2eBjdwKafbAufcFvVsczjqAe+ObcjT2FBfi6f5yYDLgqgegGuBW5/kc8bDka/YF91K3qCSNUiTCOfZF9BsF8D4QgNrF7E5XruIje+54KFULREVNu3HcQ1DUXXW47nhIKjbAgvFttA05vsEUxRM8xyDG4l93ILyq5W4mu99lKT5tBqRvm764bJg2G7YBvSRfgCTm/AKZzJfs5+9mPmtkavvqXIOfY+hL8ZqJNEsRP779/gku9SiB9+uxDe7advdS7rvxrL3xc3Fq7Buj9XNKuya/azC6jjXXoXdu/a9Q+6pQX1Tnfte6XOfI4pgcGhwvSh4ON9hnK/jDul8W+RiBnx1egEZx3Uy2mYnztdWFOiZ3ThfdRx3rByGUOoDYJ1scPmeqUXieMhEXsGsHnIJ1j2vXKbmXIHV165B8/owXZGecDzKXvr/9HjV393y4zlV0hRs6GP5sW8j9j+PNVYXS0xr5O45/61Frq9I2bqxVWuirNJ9ZsDd2wj/zrMi26xbkdOXFeWD34oVAWMynBnZtxH8XP8Apq1Jp3a0iwAAVBUIujtyadcTtc9BMBIH1VEaZO1CMhLHuwMkHj5E5bjLYKLMz5IGWZCWN4vwhtVZ8AjMzlsXVHtoewZTzf2dtTC0cG8CyKTxQeW3DHCRVzf+LwDAtJQT5ZM6ArYGAPsaALQ4nHc5AG6DL+nhAZ02p5eKTyM+ou0myb5bWeK9oPqJTyIUSr+kv7aMcY3h8GpxYPDxKcmlisxzohXeOzXWd/UhiashhTNtzkEYWIhRdkBg5HyrWQTHgCnmUNFwrpeSOUhRDWuBKPZh9CwL1jgIoiZuVmOpK6xGQDmnY2nCyyKCKatFTYaepRbNZteZoohTTWD+4auJT6MFSzkYoUvWuF0pQbNv/pgbMH/rVI4YPzNR7EkLoujCtmvoyNNELQ//pWNOr/7L0xw0/JL+ywHD+S9Ps6X5kkpwvQGVoEn4O9NWaYeAIOGsYsJSl5WmGow0/dCci/jMHs1T1ajLD0y60qNmN/QlyQQM9euR7tjEb4//SiFL+x3/X4U1/w8=</diagram></mxfile>
```

Comment:
Add newline at EOF for Travis checks

Reply:
It's there! You can't see it because Github doesn't display EOF newlines